pom.xml

```diff
@@ -14,19 +14,20 @@
   <inceptionYear>2017</inceptionYear>
 
   <properties>
+    <java.version>11</java.version>
```

Comment:
I think the default version should still be Java 8, since Spring Boot also maintains backwards compatibility to Java 8. For the Java 11 build, one could use a build property instead of manipulating the settings file: -Djava.version=11

Reply:
Done in 6bd41c1deb2a348d4f6dcee2f6e1f48c0376b6d3.